lib/notion_to_md/blocks/types.rb

```diff
@@ -108,34 +108,19 @@ def table_row(block)
         end
 
         def toggle(block)
-          result = []
-          result << "> [!CAUTION]"
-          result << "> **Unsupported Toggle Block**"
-          result << "> <br />"
-          result << "> Content needs to be imported manually."
-          result << "> <br />"
-          result << "> <br />"
-          result << "> This is the toggle title to help you find it in the original Notion page:"
-          result << "> <br />"
-          title = block[:rich_text].map {|text| Text.send(text[:type], text)}.join
-          result << "> **#{title}**"
-          result << "> <br />"
-          result.join("\n")
+          result = <<-TEXT
```

Comment:
So are we counting that the toggles won't have any child blocks inside?

Reply:
We support only text in toggles yes but it should already reduce the number of manual copy/pastes